packages/create-sitecore-jss/src/templates/nextjs/.env

```diff
@@ -9,7 +9,8 @@
 # in your Sitecore configuration (see \sitecore\config\<%- appName %>.config).
 # Be sure to update these values accordingly as your public endpoint changes.
 # See https://jss.sitecore.com/docs/fundamentals/services/view-engine
-PUBLIC_URL=http://localhost:3000
+# In production non-editing environments it is desirable to use relative urls, so this may be set to empty string
+#PUBLIC_URL=http://localhost:3000

```

Comment:
We should mention http://localhost:3000 is used by default if undefined. And just leave it empty here in the commented value?
```suggestion
# If undefined, http://localhost:3000 is used by default.
# In production non-editing environments it is desirable to use relative URLs, so this may be set to empty string.
#PUBLIC_URL=
```